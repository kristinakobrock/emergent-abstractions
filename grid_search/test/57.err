  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.51it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.28it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 22.39it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 24.39it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 25.51it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 24.68it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 24.78it/s] 50%|##############################################                                              | 25/50 [00:01<00:00, 26.03it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 23.61it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 26.32it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 28.51it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 26.35it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 25.85it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 25.67it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 24.62it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.12it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.80it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 28.27it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 23.54it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 25.68it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.73it/s]
