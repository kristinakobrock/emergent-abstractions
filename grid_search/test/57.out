# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6656377911567688, "acc": 0.6275066137313843, "length": 3.539271116256714, "mode": "train", "epoch": 1}
{"loss": 0.6770516633987427, "acc": 0.6156249642372131, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.660373330116272, "acc": 0.6280338168144226, "length": 3.562023639678955, "mode": "train", "epoch": 2}
{"loss": 0.679275393486023, "acc": 0.619921863079071, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.658654510974884, "acc": 0.6298760175704956, "length": 3.5855016708374023, "mode": "train", "epoch": 3}
{"loss": 0.6671842932701111, "acc": 0.619921863079071, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6563015580177307, "acc": 0.6305601000785828, "length": 3.6475284099578857, "mode": "train", "epoch": 4}
{"loss": 0.6601124405860901, "acc": 0.6207031607627869, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6486226916313171, "acc": 0.6319800019264221, "length": 3.7382090091705322, "mode": "train", "epoch": 5}
{"loss": 0.6382854580879211, "acc": 0.638671875, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.6268974542617798, "acc": 0.6530868411064148, "length": 3.8667492866516113, "mode": "train", "epoch": 6}
{"loss": 0.6268970370292664, "acc": 0.658203125, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.6060267686843872, "acc": 0.6742070317268372, "length": 3.9271304607391357, "mode": "train", "epoch": 7}
{"loss": 0.6389375329017639, "acc": 0.6533202528953552, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5924148559570312, "acc": 0.6831538081169128, "length": 3.9428796768188477, "mode": "train", "epoch": 8}
{"loss": 0.629936158657074, "acc": 0.6624999642372131, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.582199215888977, "acc": 0.6872311234474182, "length": 3.954434871673584, "mode": "train", "epoch": 9}
{"loss": 0.6188390254974365, "acc": 0.6650390625, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5688363313674927, "acc": 0.7000779509544373, "length": 3.9560253620147705, "mode": "train", "epoch": 10}
{"loss": 0.6166988611221313, "acc": 0.6742187142372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5574690103530884, "acc": 0.7102878093719482, "length": 3.9683375358581543, "mode": "train", "epoch": 11}
{"loss": 0.6108354330062866, "acc": 0.6847655773162842, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5482293963432312, "acc": 0.7118192315101624, "length": 3.9753942489624023, "mode": "train", "epoch": 12}
{"loss": 0.5938339829444885, "acc": 0.6939453482627869, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5294937491416931, "acc": 0.7265339493751526, "length": 3.977200984954834, "mode": "train", "epoch": 13}
{"loss": 0.5966011881828308, "acc": 0.6984375715255737, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5137622952461243, "acc": 0.7373753786087036, "length": 3.9760282039642334, "mode": "train", "epoch": 14}
{"loss": 0.5849547386169434, "acc": 0.7060547471046448, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.49550357460975647, "acc": 0.7531373500823975, "length": 3.982316732406616, "mode": "train", "epoch": 15}
{"loss": 0.5817539691925049, "acc": 0.7111328840255737, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.48672518134117126, "acc": 0.7578809857368469, "length": 3.9847793579101562, "mode": "train", "epoch": 16}
{"loss": 0.606002926826477, "acc": 0.7091796398162842, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.47631096839904785, "acc": 0.766435980796814, "length": 3.9885165691375732, "mode": "train", "epoch": 17}
{"loss": 0.5933366417884827, "acc": 0.7197265028953552, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4687098562717438, "acc": 0.7731670141220093, "length": 3.9889512062072754, "mode": "train", "epoch": 18}
{"loss": 0.5914734601974487, "acc": 0.7185547351837158, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.46505457162857056, "acc": 0.7770075798034668, "length": 3.99001145362854, "mode": "train", "epoch": 19}
{"loss": 0.5843586325645447, "acc": 0.7228515148162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4584376811981201, "acc": 0.782529354095459, "length": 3.9888455867767334, "mode": "train", "epoch": 20}
{"loss": 0.5815927386283875, "acc": 0.7251953482627869, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.44842982292175293, "acc": 0.787344753742218, "length": 3.9911890029907227, "mode": "train", "epoch": 21}
{"loss": 0.59861820936203, "acc": 0.7242187857627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.44655296206474304, "acc": 0.7906955480575562, "length": 3.9913885593414307, "mode": "train", "epoch": 22}
{"loss": 0.571127712726593, "acc": 0.7373046875, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4404684007167816, "acc": 0.7894963622093201, "length": 3.992140054702759, "mode": "train", "epoch": 23}
{"loss": 0.5602957606315613, "acc": 0.7466797232627869, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.44145044684410095, "acc": 0.7927448153495789, "length": 3.9938230514526367, "mode": "train", "epoch": 24}
{"loss": 0.5805063843727112, "acc": 0.7386718392372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4314223825931549, "acc": 0.7992550730705261, "length": 3.9923717975616455, "mode": "train", "epoch": 25}
{"loss": 0.5610290169715881, "acc": 0.7449219226837158, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.43208304047584534, "acc": 0.7988967299461365, "length": 3.993384838104248, "mode": "train", "epoch": 26}
{"loss": 0.5484253764152527, "acc": 0.744140625, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.42793840169906616, "acc": 0.7985515594482422, "length": 3.994891881942749, "mode": "train", "epoch": 27}
{"loss": 0.5399883985519409, "acc": 0.7513671517372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4256801903247833, "acc": 0.8029123544692993, "length": 3.994934558868408, "mode": "train", "epoch": 28}
{"loss": 0.534022867679596, "acc": 0.7562499642372131, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.42499735951423645, "acc": 0.8052199482917786, "length": 3.995316982269287, "mode": "train", "epoch": 29}
{"loss": 0.5436460971832275, "acc": 0.7542968392372131, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4113979935646057, "acc": 0.8099388480186462, "length": 3.9953815937042236, "mode": "train", "epoch": 30}
{"loss": 0.5335571765899658, "acc": 0.7619141340255737, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4081301689147949, "acc": 0.8108328580856323, "length": 3.9956412315368652, "mode": "train", "epoch": 31}
{"loss": 0.521453857421875, "acc": 0.7667970061302185, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4065980911254883, "acc": 0.8121526837348938, "length": 3.996656894683838, "mode": "train", "epoch": 32}
{"loss": 0.5204258561134338, "acc": 0.7671875357627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3971812427043915, "acc": 0.8133392930030823, "length": 3.9970474243164062, "mode": "train", "epoch": 33}
{"loss": 0.5105412006378174, "acc": 0.7716796398162842, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3986039459705353, "acc": 0.8187673687934875, "length": 3.9971957206726074, "mode": "train", "epoch": 34}
{"loss": 0.5121630430221558, "acc": 0.7736327648162842, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.39382749795913696, "acc": 0.8198959827423096, "length": 3.9976162910461426, "mode": "train", "epoch": 35}
{"loss": 0.5123361945152283, "acc": 0.7769531607627869, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.39354461431503296, "acc": 0.8212494254112244, "length": 3.9975361824035645, "mode": "train", "epoch": 36}
{"loss": 0.49603626132011414, "acc": 0.7769531607627869, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3856332302093506, "acc": 0.8258519768714905, "length": 3.9975244998931885, "mode": "train", "epoch": 37}
{"loss": 0.4957096576690674, "acc": 0.7761719226837158, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3821817934513092, "acc": 0.8238490223884583, "length": 3.9975361824035645, "mode": "train", "epoch": 38}
{"loss": 0.4928150177001953, "acc": 0.7767577767372131, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.37829121947288513, "acc": 0.8311613202095032, "length": 3.9973156452178955, "mode": "train", "epoch": 39}
{"loss": 0.49541395902633667, "acc": 0.7845702767372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.378938227891922, "acc": 0.8311576843261719, "length": 3.9969871044158936, "mode": "train", "epoch": 40}
{"loss": 0.49410080909729004, "acc": 0.7890625, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3630879521369934, "acc": 0.8355929851531982, "length": 3.9975969791412354, "mode": "train", "epoch": 41}
{"loss": 0.4732404947280884, "acc": 0.7925781607627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.36057668924331665, "acc": 0.8402548432350159, "length": 3.9972991943359375, "mode": "train", "epoch": 42}
{"loss": 0.47188329696655273, "acc": 0.8009766340255737, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3493782877922058, "acc": 0.8429147601127625, "length": 3.997908592224121, "mode": "train", "epoch": 43}
{"loss": 0.47412359714508057, "acc": 0.7998046875, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3471115529537201, "acc": 0.8424400687217712, "length": 3.997706651687622, "mode": "train", "epoch": 44}
{"loss": 0.4536534547805786, "acc": 0.8048828840255737, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3521423041820526, "acc": 0.845808744430542, "length": 3.9977610111236572, "mode": "train", "epoch": 45}
{"loss": 0.45484381914138794, "acc": 0.8050780296325684, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3448377549648285, "acc": 0.8457899689674377, "length": 3.997407913208008, "mode": "train", "epoch": 46}
{"loss": 0.456220805644989, "acc": 0.8033203482627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3422684669494629, "acc": 0.8474466800689697, "length": 3.997499465942383, "mode": "train", "epoch": 47}
{"loss": 0.4475221633911133, "acc": 0.8037109375, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.34390419721603394, "acc": 0.8491995930671692, "length": 3.996826171875, "mode": "train", "epoch": 48}
{"loss": 0.44990408420562744, "acc": 0.8080077767372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3356900215148926, "acc": 0.8508275747299194, "length": 3.9979066848754883, "mode": "train", "epoch": 49}
{"loss": 0.44762104749679565, "acc": 0.8041015267372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.33786696195602417, "acc": 0.855054497718811, "length": 3.9975972175598145, "mode": "train", "epoch": 50}
{"loss": 0.4576866626739502, "acc": 0.7982421517372131, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.33376869559288025, "acc": 0.8545914888381958, "length": 3.997856616973877, "mode": "train", "epoch": 51}
{"loss": 0.4325031638145447, "acc": 0.8140625953674316, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3283827006816864, "acc": 0.8536682724952698, "length": 3.9984936714172363, "mode": "train", "epoch": 52}
{"loss": 0.4419744610786438, "acc": 0.8179687857627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.32580485939979553, "acc": 0.8574868440628052, "length": 3.9977622032165527, "mode": "train", "epoch": 53}
{"loss": 0.442497193813324, "acc": 0.8119140267372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.319381445646286, "acc": 0.8613255023956299, "length": 3.9980454444885254, "mode": "train", "epoch": 54}
{"loss": 0.4432670474052429, "acc": 0.8123045563697815, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.31545451283454895, "acc": 0.862606406211853, "length": 3.999253988265991, "mode": "train", "epoch": 55}
{"loss": 0.4488416314125061, "acc": 0.8134765028953552, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.31445154547691345, "acc": 0.8639880418777466, "length": 3.997134208679199, "mode": "train", "epoch": 56}
{"loss": 0.44155851006507874, "acc": 0.8148437142372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.310714453458786, "acc": 0.8669037818908691, "length": 3.998882532119751, "mode": "train", "epoch": 57}
{"loss": 0.43814218044281006, "acc": 0.8160157203674316, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3073837161064148, "acc": 0.8679118752479553, "length": 3.999051094055176, "mode": "train", "epoch": 58}
{"loss": 0.4283013939857483, "acc": 0.8132811784744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3080102801322937, "acc": 0.8717468976974487, "length": 3.9990060329437256, "mode": "train", "epoch": 59}
{"loss": 0.43217092752456665, "acc": 0.8156251311302185, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.30559056997299194, "acc": 0.8706539869308472, "length": 3.9982635974884033, "mode": "train", "epoch": 60}
{"loss": 0.42096132040023804, "acc": 0.8179687857627869, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.717096745967865
